tests/drivers/uart/uart_async_api/src/test_uart_async.c

```diff
@@ -607,8 +646,16 @@ ZTEST_USER(uart_async_timeout, test_forever_timeout)
 		      "RX_DISABLED timeout");
 }
 
-
-ZTEST_DMEM uint8_t chained_write_tx_bufs[2][10] = {"Message 1", "Message 2"};
+#ifndef CONFIG_UART_WIDE_DATA
+ZTEST_DMEM DATA_TYPE chained_write_tx_bufs[2][10] = {"Message 1", "Message 2"};
+#else
+ZTEST_DMEM DATA_TYPE chained_write_tx_bufs[2][10] = {{0x004d, 0x0065, 0x0073, 0x0073,
+						      0x0061, 0x0067, 0x0065, 0x0020,
+						      0x0031},
+						     {0x004d, 0x0065, 0x0073, 0x0073,
+						      0x0061, 0x0067, 0x0065, 0x0020,
+						      0x0032}};
+#endif
```

Comment:
nit: maybe this can be combined into one statement too?

Reply:
Fixed